backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -397,3 +511,50 @@ def create_table_query(
                 sql_query += f"{key} {sql_type}, "
 
         return sql_query.rstrip(", ") + ");"
+
+    @staticmethod
+    def build_sql_insert_query(table_name: str, sql_keys: list[str]) -> str:
+        keys_str = ",".join(sql_keys)
+        values_placeholder = ",".join(["%s" for _ in sql_keys])
+        return f"INSERT INTO {table_name} ({keys_str}) VALUES ({values_placeholder})"
+
+    @staticmethod
+    def prepare_sql_values(cls_name: str, sql_values: Any) -> Any:
+        if cls_name == DBConnectionClass.MSSQL:
+            return tuple(sql_values)
+        return sql_values
+
+    @staticmethod
+    def execute_cursor_query(engine: Any, sql: str, sql_values: tuple[Any]) -> None:
+        with engine.cursor() as cursor:
+            cursor.execute(sql, sql_values)
+        engine.commit()
+
+    @staticmethod
+    def execute_bigquery(
+        engine: Any, table_name: str, sql_keys: list[str], sql_values: list[Any]
+    ) -> None:
+        query = f"""
+            INSERT INTO {table_name} ({', '.join(sql_keys)})
+            VALUES ({', '.join(['@' + key for key in sql_keys])})
+        """
+        query_parameters = [
+            bigquery.ScalarQueryParameter(key, "STRING", value)
+            for key, value in zip(sql_keys, sql_values)
+        ]
+        job_config = bigquery.QueryJobConfig(query_parameters=query_parameters)
+        engine.query(query, job_config=job_config)
+
+    @staticmethod
+    def execute_generic_query(engine: Any, sql: str, sql_values: Any) -> None:
+        engine.query(sql)
+
+
+class ExceptionHelper:
```

Comment:
Avoid multiple classes in same file if not necessory

Reply:
done